apa.h

```diff
@@ -77,6 +77,25 @@ typedef struct apa_toc_type
 } apa_toc_t;
 typedef /*@out@*/ /*@only@*/ /*@null@*/ apa_toc_t *apa_toc_p_t;
 
+#define PPAA_START 0x1000
+static const char *PPAA_MAGIC =
+    "PS2ICON3D";
+
+/* Parent Partition Attribute Area
+ * The header size is fixed at 512 bytes.
```

Comment:
maybe it is better to use the term: 1 HDD sector size? In theory, it is possible to use HDD with a different sector size than 512 bytes. Never saw such an HDD in real life.

Reply:
I prefer to strictly speaking of 512 bytes that refer to sector since ok that sector alignement, but i found clearer that important is struct is 512 bytes.